src/util/test/TimerTests.cpp

```diff
@@ -87,7 +87,12 @@ TEST_CASE("VirtualClock from_time_t", "[timer]")
 
 TEST_CASE("virtual event dispatch order and times", "[timer]")
 {
+    // Enabling pull mode introduces various timers (e.g.,
```

Comment:
you can leave this for later, but pull mode is off by default already (and I am not sure this is the only place where things will break when enabled by default)